src/scales/scale.time.js

```diff
@@ -313,6 +315,26 @@ function generate(min, max, minor, major, capacity, options) {
 		last.add(1, minor);
 	}
 
+	// for series we want to include only user supplied timestamps
+	if (options.distribution === 'series') {
```

Comment:
[suggestion] since this block is it's own thing. It might make sense to factor this out and then call `generate` or `generateSeries` as applicable. @simonbrunel thoughts?